src/test/java/io/epirus/console/project/InteractiveOptionsTest.java

```diff
@@ -27,6 +27,7 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class InteractiveOptionsTest {
+
```

Comment:
nit: newline :D

Reply:
this is the core change I am making in this PR - I would like to keep this change